### PR TITLE
Home

### PR DESCRIPTION
**Pull request by simrin051**
Files To Check

1. Home.jsx (/pages/home/home.jsx)
2. Components folder (/src/components)

### CODE REVIEW

File: src/pages/home/home.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import wallpaper from './wallpaper.jpg';
+import { CategoryList } from './../categories/CategoryList';
+
+export const Home = () => {
+    return (
+        <div>
+            <div className="adv-container">
+                <img src={wallpaper} alt="wallpaper not loaded" />
```
Comment (mikesch803) [inline src/pages/home/home.jsx:8]: alt should be describing the actual image

File: src/components/Buttons/AddToCart.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+const AddProductToCart = (product, productDispatch) => {
+    AddToCart(product, productDispatch);
+}
+
+const isAlreadyAddedToCart = (itemsInCart, product) => {
+    if (itemsInCart) {
+        return itemsInCart.find((item) => item._id === product._id);
+    }
+    return false;
+}
```
Comment (mikesch803) [inline src/components/Buttons/AddToCart.jsx:17]: as you have extracted this function, you can move this function to utils folder

File: src/components/Buttons/AddToWishlist.jsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import './Buttons.css';
+
+export const AddToWishlist = () => {
+    return (
+        <button className="like-btn" onClick={() => { AddToWishlist() }}>
```
Comment (mikesch803) [inline src/components/Buttons/AddToWishlist.jsx:5]: here i m not able to understand you are calling the same the component

File: src/components/Card/CategoryCard.jsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    const {
+        _id,
+        categoryName,
+        description,
+        image
+    } = category;
+    return (<div class="category">
+        <Link to="/products">
+            <img src={image} />
+            <div className="topicname">{category.categoryName}</div>
```
Comment (mikesch803) [inline src/components/Card/CategoryCard.jsx:13]: you have already destructured category, you can use categoryName here

File: src/components/NavBar/NavBar.jsx [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+    var sidebar = document.getElementById('sidebar');
+    var display = document.getElementById('sidebar').style.display;
+    sidebar.style.display = (display == 'block') ? 'none' : 'block';
+    if (sidebar.style.display == 'block') {
+        document.getElementById("menu-icon").style.display = 'none';
+        document.getElementById("logo").style.display = 'none';
+    } else {
+        document.getElementById("menu-icon").style.display = 'block';
+        document.getElementById("logo").style.display = 'block';
```
Comment (mikesch803) [inline src/components/NavBar/NavBar.jsx:16]: you are manipulating the dom, you can show the sidebar without doing manipulating it

**Review by mikesch803 — CHANGES_REQUESTED** [review]
hey I put some suggestions, and one more thing you should share the preview

File: src/components/Buttons/AddToCart.jsx (hunk 2 above)
Comment (simrin051) [reply src/components/Buttons/AddToCart.jsx:17]: I don't have a utils folder.

File: src/components/Buttons/AddToWishlist.jsx (hunk 3 above)
Comment (simrin051) [reply src/components/Buttons/AddToWishlist.jsx:5]: removed this click

File: src/components/Card/CategoryCard.jsx (hunk 4 above)
Comment (simrin051) [reply src/components/Card/CategoryCard.jsx:13]: Done

File: src/components/NavBar/NavBar.jsx (hunk 5 above)
Comment (simrin051) [reply src/components/NavBar/NavBar.jsx:16]: This is they way used in most applications. Thanks for the suggestion.

File: src/pages/home/home.jsx (hunk 1 above)
Comment (simrin051) [reply src/pages/home/home.jsx:8]: Done

**Pull request merged**